<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1625</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="maximumSize">
      <size>
       <width>1267</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>1603</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>                                                    普通消息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                    重要消息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                    关键消息</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>                                                    消息类型                   </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>                                                      分级指控</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                      集中指控</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                      mesh指控（贪婪路由）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                      mesh指控（冗余路由）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                      mesh指控（节点移动）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                      全部</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>                                                   指控模式</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_3">
     <item>
      <property name="text">
       <string>                                                            成功率</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>                                                             通信</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
